.gitignore

```diff
@@ -1,16 +1,15 @@
-# Intellij files
-#IDE
-.idea/
-**/*.iml
+target
```

Comment:
The recommended `.gitignore` file for Jenkins plugins from the Jenkins archetype.